skopt/callbacks.py

```diff
@@ -176,3 +178,28 @@ def _criterion(self, result):
                 return False
         else:
             return None
+
+
+class DeltaYStopper(EarlyStopper):
+    """Stop the optimization if the `n_best` minima are within `delta`
+
+    Stop the optimizer if the absolute difference between the `n_best`
+    objective values is less than `delta`.
+    """
+    def __init__(self, delta, n_best=5):
```

Comment:
Is `n_best=2` a better default? That is stop when the new point is in delta of the previous point.